inventory/README.md

```diff
@@ -130,523 +151,119 @@ To build the application, we used maven build. Maven is a project management too
-
-If it is a success, you will see the below output.
-
-```
-Successfully built 68h2dy63j9bd
-Successfully tagged bc-inventorydb:v1.0.0
-```
-Then run ` cd ../inventory`
-
-Run the helm chart as below.
-
-`helm install --name=inventorydb chart/inventorydb`
-
-6. Make sure your deployment is ready. To verify run this command and you should see the availability.
-
-`kubectl get deployments`
-
-Yow will see message like below.
-
-```
-NAME                                        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-bluecompute-inventorydb                     1         1         1            1           2m
-```
-
-#### Set Up MYSQL on IBM Cloud Private
-
-1. Your [IBM Cloud Private Cluster](https://www.ibm.com/cloud/private) should be up and running.
-
-2. Log in to the IBM Cloud Private. 
-
-<p align="center">
-    <img src="https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/microprofile/static/imgs/icp_dashboard.png">
-</p>
-
-3. Go to `admin > Configure Client`.
-
-<p align="center">
-    <img src="https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/microprofile/static/imgs/client_config.png">
-</p>
-
-4. Grab the kubectl configuration commands.
-
-<p align="center">
-    <img src="https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/microprofile/static/imgs/kube_cmds.png">
-</p>
-
-5. Run those commands in your terminal.
-
-6. If successful, you should see something like below.
-```
-Switched to context "xxx-cluster.icp-context".
-```
-7. Run the below command.
-
-`helm init --client-only`
-
-You will see the below
+1. Build the docker image.
 
-```
-$HELM_HOME has been configured at /Users/user@ibm.com/.helm.
-Not installing Tiller due to 'client-only' flag having been set
-Happy Helming!
-```
+`docker build -t mysql .`
 
-8. Verify the helm version
+2. Run the container.
 
-`helm version --tls`
+`docker run -p 9041:3306 -d --name mysql -e MYSQL_ROOT_PASSWORD=password mysql`
 
-You will see something like below.
-
-```
-Client: &version.Version{SemVer:"v2.7.2+icp", GitCommit:"d41a5c2da480efc555ddca57d3972bcad3351801", GitTreeState:"dirty"}
-Server: &version.Version{SemVer:"v2.7.2+icp", GitCommit:"d41a5c2da480efc555ddca57d3972bcad3351801", GitTreeState:"dirty"}
-```
-
-9. Now run the docker build.
+3. It creates `items` table and loads the sample data.
 
 ```
 cd ..
-cd mysql
+cd inventory
 ```
 
-`docker build -t bc-inventorydb:v1.0.0 .`
-
-If it is a success, you will see the below output.
+In this case, your jdbcURL will be 
 
 ```
-Successfully built h9u2cf24d7ad
-Successfully tagged bc-inventorydb:v1.0.0
+export jdbcURL=jdbc:mysql://localhost:9041/inventorydb?useSSL=false
+export dbuser=root
+export dbpassword=password
 ```
 
-Then run ` cd ../inventory`
-
-10. Tag the image to your private registry.
-
-`docker tag bc-inventorydb:v1.0.0 <Your ICP registry>/bc-inventorydb:v1.0.0`
-
-11. Push the image to your private registry.
+### Setting up Zipkin
```

Comment:
We should explain here why we would need Zipkin. If I recall correctly, it's mostly a dashboard that displays our tracing information. I mean, it's not necessary to start Zipkin unless we want to test out our open tracing features, right?

Reply:
Sure, I can link it to the parent repo which explains why we use Zipkin. Yes like you said, running Zipkin container is not mandatory. But we have to set the host and port to default values or else we will hit this error `ERROR   ] CWMOT0001E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method failed with Exception.` since the usr feature errors out. 

So will it be nice just to say ... okay set your zipkin values to defaults and if you want to access the traces using Zipkin then follow the instructions ?